do_freebsd.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -xve
+NPROC=`sysctl -n hw.ncpu`
+
+if [ x"$1"x = x"--deps" ]; then
```

Comment:
the reason why we need `[ x$1 = x"foo" ]` is that we don't have `$1` quoted. iiuc, the old school shell looks like:
```sh
if [ x$1= x--deps ]; then
```